corehq/apps/smsbillables/migrations/0005_add_zero_http_gateway_fee_prep.py

```diff
@@ -0,0 +1,23 @@
+# -*- coding: utf-8 -*-
+from __future__ import unicode_literals
+
+from django.db import migrations
+
+from corehq.apps.sms.models import OUTGOING
+from corehq.apps.smsbillables.management.commands.add_zero_http_gateway_fee import \
+    add_zero_http_gateway_fee
+
+
+def add_zero_http_gateway_fee_prep(apps, schema_editor):
+    add_zero_http_gateway_fee('635b73026a9a8eab55c77a84bccf6be2', OUTGOING, apps)
```

Comment:
I don't think we want to tie 0 fees to this backend itself, I think I'd prefer to tie 0 fees to the HTTP api id so that it starts off at 0 for all of them and then we can override for specific ones.